app.rb

```diff
@@ -5,5 +5,13 @@ class Chitter < Sinatra::Base
     'Test page'
   end
 
+  get '/chitter' do
+    erb :chitter
+  end
```

Comment:
What do you think about replacing '/chitter' with '/'? This would mean that the user would directly see your 'Chitter feed' title without having to explicitly put that route into the URL.

Reply:
Good job with this challenge. I would suggest you look at RESTful routes. See below for more details:

get '/peeps' -> is for listing out all peeps
post '/peeps' -> is for creating a peep
get '/peeps/1' -> is for viewing the peep with an id of 1
get '/peeps/new' -> would return the form for creating a new peep
get '/peeps/1/edit' -> would return a form for editing the peep with an id of 1
put '/peeps/1' -> would update the peed with an id of 1
delete '/peeps/1' -> would delete the peep with an id of 1

More info here: https://github.com/makersacademy/course/blob/main/pills/rest.md